server/models.py

```diff
@@ -44,8 +49,53 @@ class Organization(BaseModel):
     name = Column(String(200), nullable=False, unique=True)
 
 
-def get_or_404(model: Type[Base], primary_key: str):
-    instance = model.query.get(primary_key)
-    if instance:
-        return instance
-    raise NotFound(f"{model.__class__.__name__} {primary_key} not found")
+class AdminUser(BaseModel):
+    id = Column(String(200), primary_key=True)
+    email = Column(String(200), unique=True, nullable=False)
+    external_id = Column(String(200), unique=True)
```

Comment:
don't see use case for this, but fine if you leave it.

Reply:
i only put it in because i thought you cared about it. i don't care